src/crdt.rs

```diff
@@ -32,6 +32,9 @@ use std::sync::{Arc, RwLock};
 use std::thread::{sleep, Builder, JoinHandle};
 use std::time::Duration;
 use streamer::{BlobReceiver, BlobSender, Window};
+use timing::timestamp;
+
+const GOSSIP_SLEEP_MILLIS: u64 = 100;
```

Comment:
Can this be a `Duration` and then name it something more intuitive like VALIDATOR_HEARTBEAT_DURATION or VALIDATOR_TIMEOUT?

Reply:
duration is a pia to do compute on or declare. id rather use millis

Reply:
I think gossip sleep is actually more descriptive, since its specific to membership in the gossip network, and not to what is using it.

Reply:
ok